test/e2e/e2e_test.go

```diff
@@ -402,6 +402,35 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 
 		framework.ExpectNoError(<-errChan)
 	})
+
+	ginkgo.It("should provide Internet connection continuously when all pods on master node are killed", func() {
+		ginkgo.By("Running container which tries to connect to 8.8.8.8 in a loop")
+
+		podChan, errChan := make(chan *v1.Pod), make(chan error)
+		go checkContinuousConnectivity(f, "", "connectivity-test-continuous", "8.8.8.8", 53, 30, podChan, errChan)
+
+		testPod := <-podChan
+		framework.Logf("Test pod running on %q", testPod.Spec.NodeName)
+
+		time.Sleep(5 * time.Second)
+
+		podClient := f.ClientSet.CoreV1().Pods("")
+
+		podList, _ := podClient.List(metav1.ListOptions{})
+		for _, pod := range podList.Items {
+			if pod.Spec.NodeName == "ovn-control-plane" && pod.Name != "connectivity-test-continuous" && pod.Name != "etcd-ovn-control-plane" {
```

Comment:
dont you want to kill the active master? What if the test is running HA?

Reply:
yes, that need to be address. Test were failing with nonHA scenario, so wanted to resolve that first. Working on adding the HA scenario, but seems like some of the existing tests are broken, because of the name we use for master nodes in case of HA.

Reply:
> but seems like some of the existing tests are broken, because of the name we use for master nodes in case of HA.
the ovn master election depends on the leader election performed in each master pod, not on the name of the nodes, ... I maybe wrong but don´t see the connection

Reply:
what we want to guarantee is that the connection works if the ovn master doesn´t work, right?

Reply:
As discussed, i think we need to cleanup this part.